private_sharing/models.py

```diff
@@ -395,6 +395,7 @@ class Meta:  # noqa: D101
             "/direct-sharing/oauth2-setup/#setup-oauth2-authorization"
         ),
         verbose_name="Redirect URL",
+        blank=True,
```

Comment:
Is this needed? Or maybe it's leftover from realizing we needed to specify the redirect_url...

Reply:
It's a leftover from having to specify a redirect_url.